### PR TITLE
chore(package): update dependencies

### PR DESCRIPTION
**Pull request by nolanlawson**
Unfortunately we can't upgrade to TypeScript v4 yet because that breaks something in [sfdx-core](https://github.com/forcedotcom/sfdx-core). But we can upgrade the other stuff.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,14 +25,14 @@
     "@types/chai": "4.2.11",
     "@types/jsforce": "^1.9.18",
     "@types/mocha": "^7.0.2",
-    "@types/node": "^13.13.2",
+    "@types/node": "^16.0.2",
```
Comment (ravijayaramappa) [inline package.json:28]: This dependency is also being updated in the other PR(https://github.com/salesforce/sfdx-plugin-lwc-test/pull/30/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R28)

**Comment by ravijayaramappa** [comment]
All the three PRs are touching yarn.lock and might run into merge conflicts. May be combine the updates into one PR would eliminate some of this overhead.

**Review by pmdartus — APPROVED** [review]

**Comment by nolanlawson** [comment]
@ravijayaramappa I did multiple PRs to try to isolate what's breaking. But yes it could all be one big PR.

**Comment by nolanlawson** [comment]
Combining #30 and #33 into this PR for one big mega-dep-update.

**Pull request merged**